src/services/user.service.js

```diff
@@ -1,6 +1,7 @@
 const httpStatus = require('http-status');
-const { User } = require('../models');
+const { User, Class } = require('../models');
 const ApiError = require('../utils/ApiError');
+const logger = require('../config/logger');
```

Comment:
dont need this

Reply:
ohh, it's only for debug. Forget to delete